packages/framework/src/Command/CreateApplicationDirectoriesCommand.php

```diff
@@ -64,19 +56,15 @@ public function __construct(
         $defaultPublicDirectories,
```

Comment:
They are nullable so it would require changing their order ... also it would be difficult to find them all and I would like to move this PR further. Feel free to create another one to fix not automized cases :)